lib/user.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './database_connection'
+require 'bcrypt'
+
+class User
+  attr_reader :id, :fullname, :username, :email
```

Comment:
I like the names you've chosen, particularly good for distinguishing between fullname and username